portfolio/src/main/webapp/index.html

```diff
@@ -6,22 +6,19 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+<body>
```

Comment:
body tag is misaligned. please align by indenting body tag over by two spaces!

https://google.github.io/styleguide/htmlcssguide.html#Indentation